en/docs/security-announcements/cve-justifications/2025/CVE-2025-47889.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "CVE-2025-47889"
+category: security-announcements
+date: May 14 2025
+---
+
+# CVE-2025-47889
+
+<p class="doc-info">WSO2 Products impacted: no</p>
+<p class="doc-info">Customers actions required: no</p>
+---
+
+### REPORTED VULNERABILITY
+
+Authentication claims are accepted without validation by the "WSO2 Oauth" security realm, allowing unauthenticated attackers to log in to controllers using this security realm using any username and any password, including usernames that do not exist.
+
+### REPORTED PRODUCTS
+
+- WSO2 Identity Server : 5.10.0, 5.11.0, 6.0.0, 6.1.0, 7.0.0, 7.1.0
+
+### WSO2 JUSTIFICATION
+
+As part of our analysis, we conducted a comprehensive review of WSO2 Identity Server’s usage of wso2id-oauth-plugin, including static code analysis and runtime dependency tracing. Our investigation confirmed that the wso2id-oauth-plugin, which is the specific component that could be exploited through this vulnerability, is neither referenced nor invoked anywhere in the WSO2 Identity Server codebase. Therefore, we conclude that this vulnerability does not pose a security risk to WSO2 Identity Server products.
```

Comment:
```suggestion
As part of our analysis, we conducted a comprehensive review of WSO2 Identity Server’s usage of wso2id-oauth-plugin, including static code analysis and runtime dependency tracing. Our investigation confirmed that the `wso2id-oauth-plugin`, which is the specific component that could be exploited through this vulnerability, is neither referenced nor invoked anywhere in the WSO2 Identity Server codebase. Therefore, we conclude that this vulnerability does not pose a security risk to WSO2 Identity Server products.
```

@ShanChathusanda93 Shouldn't we mention that this WSO2 Oauth plugin is a third party plugin developed jenkins to authenticate with wso2is?

Reply:
Added extra details